code/modules/gear_presets/survivors/solaris/crashlanding-offices_insert_bigred.dm

```diff
@@ -73,6 +75,24 @@
 	new_human.equip_to_slot_or_del(new /obj/item/stack/sheet/plasteel/med_small_stack(new_human), WEAR_IN_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(new_human), WEAR_R_HAND)
 
+	..()
+// /obj/effect/landmark/survivor_spawner/bigred_crashed_pmc_leader
+
+/datum/equipment_preset/survivor/pmc/pmc_leader
+	name = "Survivor - PMC Leader"
+	assignment = JOB_PMC_LEADER
+	rank = JOB_PMC_LEADER
+	paygrades = list(PAY_SHORT_PMC_TL = JOB_PLAYTIME_TIER_0)
+	role_comm_title = "SL"
+	skills = /datum/skills/civilian/survivor/pmc/lead
+
+/datum/equipment_preset/survivor/pmc/pmc_leader/load_gear(mob/living/carbon/human/new_human)
```

Comment:
Could add the command version of the PMC headset for the Leader. The hvh version to be specific (to avoid having access to marine comms)
/obj/item/device/radio/headset/distress/pmc/command/hvh

Reply:
I just went ahead and did it for all of them. Also decided to change the actual factions so they all show up under WY tab instead of a mix of surv, synth, and "human". Rank icons work now too. Gave the surv supervisor a PMC headset so he can see the effects of the orders he gives as well as rank icons. The Corporate manager preset he's based off of gets that headset as well, apparently.